typings.d.ts

```diff
@@ -0,0 +1,139 @@
+declare module 'resin-components' {
+	import { Component } from 'react';
+	import * as ResinSdk from 'resin-sdk';
```

Comment:
Is this a 'single space' indentation?

Reply:
No, it uses tabs

Reply:
I swear that Chrome rendered it like a single space yesterday... Nevermind & sorry for the noise.

Reply:
I swear that Chrome rendered it like a single space yesterday... Nevermind & sorry for the noise.